src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor(){
+        String[] color = new String[]{"Black", "Yellow", "Orange", "White", "Purple", "Blue", "Green", "Pink"};
```

Comment:
Consider defining the array of colors as a constant field. This follows the best practice of using constants for fixed sets of values and allows for easier changes and maintenance.